lib/specinfra/command/base/user.rb

```diff
@@ -1,15 +1,17 @@
 class Specinfra::Command::Base::User < Specinfra::Command::Base
   class << self
     def check_exists(user)
-      "id #{escape(user)}"
+      "getent passwd #{escape(user)}"
     end
 
     def check_belongs_to_group(user, group)
-      "id #{escape(user)} | sed 's/ context=.*//g' | cut -f 4 -d '=' | grep -- #{escape(group)}"
+      "getent group #{escape(group)} | grep -- #{escape(user)}"
     end
```

Comment:
Thanks! But this seems not to be backward compatible.

For example, the result of 

```
id pollinate | sed 's/ context=.*//g' | cut -f 4 -d '=' | grep -- daemon
```

is different from the result of

```
getent group daemon | grep -- pollinate
```

on Ubuntu 18.04. Please check it.